rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -86,7 +94,6 @@ def reduce_mean_valid(t):
 
     if policy.config["use_gae"]:
         prev_value_fn_out = train_batch[SampleBatch.VF_PREDS]
-        value_fn_out = model.value_function()
```

Comment:
One reason for the new API is that we can get rid of these async `value_function` calls.